### ISSUE TITLE
Add ability to  

### ISSUE THREAD

**Issue by freelancesam** [wulkano/kap#150]
**Platform:**

OSX

**Kap Version:**
1

### Steps to reproduce

Command + Shift + 5

### Current behaviour

Shortcut is clashing with another shortcut with no way to change it.

### Expected behaviour

Preferences should have an option to set new shortcut

**Comment by CodeTheory** [comment]
Hey @freelancesam, thanks for the issue.

This is planned in an upcoming release. We'll use this issue to keep you up to date.

**Comment by controversial** [comment]
Command + Shift + 5 happens to be the "run" command in Codepen

**Comment by CodeTheory** [comment]
Contributions are welcome 😄

**Comment by boaticus** [comment]
Also Skitch screenshot is Command + Shift + 5.

Perhaps I'll have a look and submit a PR.

**Comment by fokusferit** [comment]
I'd like to work on this too. 
Does it make sense to put it only in to the preferences Window or also on the control panel ?

**Comment by skllcrn** [comment]
That's awesome news @fokusferit! Keyboard shortcut configuration would live within preferences in it's own tab.

**Comment by onordbo** [comment]
I must agree with @skllcrn here. I don't see the interaction value in investing scarce interface real-estate in order to enable direct access to manipulate keyboard shortcuts from the control panel.

**Comment by FezVrasta** [comment]
I just wanted to share my small experience, I tried to work on this feature but I'm not confident at all with the code base.

If it used something like React or Vue it would have been much easier, and I guess you'd get more contributors. Just my 2cents

**Comment by CodeTheory** [comment]
@FezVrasta We absolutely want to rewrite Kap with React ([Next.js](https://leo.im/2017/electron-next)) but we wanted our feature set to take priority so we're currently working on getting v2 out and then we'll absolutely work on porting Kap into React ❤️ 

Thanks for your input.

**Comment by skllcrn** [comment]
Until we ship this I suggest adding a setting to turn keyboard shortcuts on or off all together, that way it's easy for people to avoid conflicts with other apps.

**Comment by YoavGivati** [comment]
Would really love some customizability here. Even if kap just looked for some config file that we can edit in a text editor before falling back to the hard-coded default.

(whether the config file is something we create in our user directory, or something we need to find after each update in Applications -> Kap (right click -> show package contents) -> some folder.

**Comment by FezVrasta** [comment]
To me it would be enough to have a checkbox to disable them

**Comment by skllcrn** [comment]
In progress as part of #450 

![configure-plugin](https://user-images.githubusercontent.com/8440026/39950972-d1565108-5585-11e8-967a-93b06686f7a0.png)

**Comment by skllcrn** [comment]
[`globalShortcut`](https://electronjs.org/docs/api/global-shortcut#globalshortcutregisteraccelerator-callback)

> When the accelerator is already taken by other applications, this call will silently fail. This behavior is intended by operating systems, since they don't want applications to fight for global shortcuts. 

We'll need a hard-coded list of shortcuts and messages. Could be a good reusable Electron module!

**Comment by skllcrn** [comment]
#507

**Comment by jesset55** [comment]
Hi!  That latest OSX release Mojave has made the default hotkey for screenshots command+shift+5 which interferes with Skitch's default hotkey.  I need the ability to change/customize Skitch's hotkeys to avoid this issue.  Please let me know when that will be done :)

**Comment by skllcrn** [comment]
Hey @jesset55, You can sign up to get early access and start using the new custom keyboard shortcuts at [getkap.co](https://getkap.co)!
